src/backend/marsha/core/api/video.py

```diff
@@ -585,11 +590,7 @@ def start_sharing(
         video.save()
         serializer = self.get_serializer(video)
 
-        broadcast_message(
-            str(video.id),
-            "start_sharing",
-            json.dumps(serializer.data, cls=DjangoJSONEncoder),
-        )
+        channel_layers_utils.dispatch_video_to_groups(video)
```

Comment:
Don't you think we should specify an event type here ?
```suggestion
        channel_layers_utils.dispatch_video_to_groups("start_sharing", video)
```
With `dispatch_video` sending something like 
```
    async_to_sync(channel_layer.group_send)(
        room_name.format(video_id=str(video.id)),
        {"type": event_type, "video": serialized_video.data},
    )
```

Reply:
No because we just want the updated video, not the action leading to this change. In the front application there is no need about this information but just to know that the video is updated and the store should be updated. Once updated all the component tree should be updated.